packages/main/src/themes/base/sizes-parameters.css

```diff
@@ -144,6 +144,15 @@
 	/* SplitButton */
 	--_ui5_split_button_middle_separator_top: 0;
 	--_ui5_split_button_middle_separator_height: 2.25rem;
+
+	/* Toolbar */
+	--_ui5-toolbar-separator-height: 2rem;
+
+	--_ui5-toolbar-padding-left: 0.5rem;
```

Comment:
if variables -_ui5-toolbar-padding-*, --_ui5-toolbar-item-* are not subject to compact/cozy mode, they should not be defined in this file. For example --_ui5-toolbar-separator-height is 2rem in cozy and 1.5rem in compact, what about the rest?